ee/clickhouse/models/property.py

```diff
@@ -233,13 +233,17 @@ def prop_filter_json_extract(
         assert isinstance(prop.value, str)
         prop_value_param_key = "v{}_{}".format(prepend, idx)
 
+        if prop.key in EVENT_ATTRIBUTE_RESERVED_PROPERTIES_BY_TYPE["DateTime"]:
```

Comment:
Does anything need to be done here about checking prop.key != 'event'

Reply:
Doh, great catch... Added test case to force that out